manifests/database/postgresql.pp

```diff
@@ -65,7 +65,7 @@
   }
 
   exec { 'update_pgpass':
-    command => "echo ${database_host}:5432:${database_name}:${database_user}:${database_password} >> /root/.pgpass",
+    command => "echo \"${database_host}:5432:${database_name}:${database_user}:${database_password}\" >> /root/.pgpass",
```

Comment:
This will shoke if the password has `"` char, right?  `shell_escape()` from stdlib should be safer in this case.

Reply:
Thanks, I was not aware of this function. I just updated the Pull Request.